### PR TITLE
testing/gdal: add py3-gdal subpackage and upgrade to 2.4.2

### PR DESCRIPTION
**Pull request by zarch**
Hi, 
this is my first contribution to aports, I've read the documentation on how to contribute, but please let me know if something is wrong and/or I've missed something. :-)

The commit add the subpackage: `py3-gdal` to be able to use gdal from `python3`.

I have also updated the version from `2.4.0` to `2.4.2`.

Regarding the version, there is also available the version: `3.0.1` released 2019-06-28, see the [changelog](https://github.com/OSGeo/gdal/blob/v3.0.1/gdal/NEWS), but I'm not sure about the general alpine distribution policy, it is a major release, so it might break other packages.

### CODE REVIEW

File: testing/gdal/APKBUILD [hunk 1]
```diff
@@ -36,26 +38,43 @@ build() {
...
+	pkgdesc="$pkgname (python2 bindings)"
 
-	cd "$builddir"/swig/python
+	cd "$builddir"/swig/python2
 	python2 setup.py install --prefix=/usr --root="$subpkgdir"
 
 	chmod a+x scripts/*
 	install -d "$subpkgdir"/usr/bin
 	install -m755 scripts/*.py "$subpkgdir"/usr/bin/
 }
```
Comment (Cogitri) [inline testing/gdal/APKBUILD]: Just drop python2 in favour of python3, we're in the process of removing py2 anyway since it'll be EOL soon.

**Comment by Cogitri** [comment]
LGTM to me otherwise, you don't have to do the major bump if you don't feel like it :)

File: testing/gdal/APKBUILD (hunk 1 above)
Comment (tclarke) [reply testing/gdal/APKBUILD]: Agreed, just remove all the py2 stuff.

**Comment by Cogitri** [comment]
Uhh... that doesn't look right

**Comment by zarch** [comment]
> Uhh... that doesn't look right

mmh... yes I see... I did a `git pull --rebase` but this have added all the other commits on this PR.
Should I close the PR do my changes and open a new PR? Other better idea on how I can solve this?

**Comment by Cogitri** [comment]
Hm, usually rebasing upon alpine master should work just fine, no clue what went wrong here

**Comment by Cogitri** [comment]
Ah, you rewrote history. Put your two commits at the end of the history (to not put it in front of the already existing ones) and squash them into one please.

**Comment by zarch** [comment]
@Cogitri, I did a new PR #11555 where you can find just one single commit. I close this PR.
Sorry for the noise.

**Pull request closed without merging**